src/main/scala/com/github/http4sbin/http/services/GetService.scala

```diff
@@ -0,0 +1,30 @@
+package com.github.http4sbin.http.services
+
+import org.http4s.HttpService
+import org.http4s.dsl._
+
+import argonaut._, Argonaut._
+
+import com.github.http4sbin.http.Http4sHelpers.{getHeadersAsJson, getQueryAsJson, getAbsoluteURI}
+
+object GetService {
+
+  val service = HttpService {
+    case req @ GET -> Root / "get" => {
+      val ip = req.remote match {
+        case Some(ipAddress) => jString(ipAddress.getAddress.toString)
+        case None            => jNull
+      }
+      val headers = getHeadersAsJson(req.headers)
+      val args = getQueryAsJson(req.uri.query)
+      val url = getAbsoluteURI(req)
+
+      Ok(Json(
+        "args"    := args,
+        "origin"  := ip,
+        "headers" := headers,
+        "url"     := url
+      ).toString)
```

Comment:
Why `.toString`? If you return call `Ok`, with `Json`, it uses the `EntityEncoder` for `Json` to serialize the response, which adds `application/json` to the content-type. If you call `Ok` on a `String`, you use the `EntityEncoder` for `String`, and I THINK... it will use an plain text content-type, which might not be what you want.